src/google-maps/google-map/google-map.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
+import {ReplaySubject} from 'rxjs';
+import {take} from 'rxjs/operators';
+
+/**
+ * Angular component that renders a Google Map via the Google Maps JavaScript
+ * API.
+ * @see https://developers.google.com/maps/documentation/javascript/reference/
+ */
+@Component({
+  selector: 'google-map',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<div #map></div>',
+})
+export class GoogleMap implements OnInit {
+  @Input() height = '500px';
+  @Input() width = '500px';
+  // TODO(mbehrlich): add options, handlers, properties, and methods.
+
+  @ViewChild('map', {static: true}) set mapEl(mapEl: ElementRef) {
+    this._mapEl$.next(mapEl);
+  }
+
+  private readonly _mapEl$ = new ReplaySubject<ElementRef>(1);
+  private readonly _map$ = new ReplaySubject<google.maps.Map>(1);
+
+  ngOnInit() {
+    // default options for now
+    const options: google.maps.MapOptions = {
+      center: {lat: 50, lng: 50},
```

Comment:
Do we have an easter egg opportunity to set the default lat/lon to somewhere interesting?

Reply:
Good idea, I've set it to the Colosseum for now, because you can actually see that on the map. Let me know if you have a better idea.

Reply:
I was just going to suggest something like 1600 Amphitheater or 111 8th Ave, but any major landmark seems fine to me

Reply:
That makes more sense. Done.